devex/vsextension/vsix-release-process.md

```diff
@@ -0,0 +1,78 @@
+# VS Extension Release Process
+
+The process used to build and release the nuget package from these
+sources is as follows.  Currently the signature on the release is
+a Microsoft signature generated using the Microsoft [PODSS](https://aka.ms/podss) process.
+In the future, this process will change to a use a non-Microsoft
+signature once another signing process is defined for Open Enclave.
+
+1. Build the latest [Cross-Platform Open Enclave SDK NuGet Package](https://github.com/HernanGatta/openenclave-cross-plat-nuget).
+   Make a note of the new version number for use in step 3.
+
+2. Make a note of the old version referenced in the VS extension sources.
+   This can be found by running the following in the vsextension directory:
+
+      ```cmd
+      findstr open-enclave-cross ProjectWizard\source.extension.vsixmanifest
+      ```
+
+3. Update the open-enclave-cross version referenced in the VS extension
+   sources. For example, from a bash prompt, to update from
+   0.8.1-c3b6262c-3 to 0.11.0-rc1-cbe4dedc-1:
+
+      ```bash
+      grep -RiIl '0.8.1-c3b6262c-3' | xargs sed -i 's/0.8.1-c3b6262c-3/0.11.0-rc1-cbe4dedc-1/g'
+      ```
+
+4. Bump the Identity Version number in the ProjectWizard\source.extension.vsixmanifest
```

Comment:
It is unclear to me how the nuget package built from Hernan's PR ties in here. Should it be placed somewhere before VSExtension.sln is built?

Reply:
Good point, this was in the README.md in the previous commit of this PR and it got removed, so let me put that part back in!   I'll put it in this md rather than back in the README.md though.

Reply:
Updated, now is the new step 2 in the release process doc.

Reply:
Thanks.